Modules/Sources/WordPressUI/Views/Users/Components/UserProfileImage.swift

```diff
@@ -0,0 +1,37 @@
+import SwiftUI
+
+struct UserProfileImage: View {
```

Comment:
Please, use `AvatarView` and avoid `AsyncImage` as it uses `URLSession.shared` and basically has no caching. `AvatarView` is in the app target, so `User*` types will likely need to be moved to the app target.

The app has `CachedAsyncImage` that uses app's `ImageDownloader` with a memory cache.